src/server/controllers/tasks.js

```diff
@@ -52,4 +52,27 @@ export class TaskManager {
       }
     });
   }
+  /**
+         *
+         *Updating a task
+         * @memberof TaskManager
+         */
+  updateTask() {
+    Task.findById(this.req.params.id).exec((error, tasks) => {
+      if (error) {
+        this.res.json({
+          error: error.message,
+        });
+      } else {
+        if (this.req.body.name && this.req.body.body) {
+          tasks.name = this.req.body.name || tasks.name;
+          tasks.body = this.req.body.body || tasks.body;
+          tasks.save();
+          this.res.json(tasks);
+        } else {
+          this.res.json({msg: 'you have an error please check name and body keys'});
```

Comment:
What error? Be specific.

Reply:
will do that

Reply:
I have decided to do the validations in models/tasks.js since the client wont be able to change the name of the keys or the form labels